addons/hr/tests/test_hr_employee.py

```diff
@@ -161,6 +161,33 @@ def test_employee_member_of_department(self):
             employees.filtered_domain(['!'] + employees._search_part_of_department('in', [True])),
             emp_other + emp_parent)
 
+    def test_batch_user_creation_skips_employees_with_existing_users_email(self):
+        existing_email_employee, new_employee = self.env['hr.employee'].create([
+            {
+                'name': 'Duplicate Email Employee',
+                'work_email': self.user_without_image.email,
```

Comment:
By formatted email I mean something like
```suggestion
                'work_email': f'"Some Other Name" <{self.user_without_image.email_normalized}>',
```

Reply:
Could also test a multi email (you never know what people do with the inputs), like
```suggestion
                'work_email': '"Name1" <name@test.example.com>, "Name 2" <name2@test.example.com>',
```